lms/services/exceptions.py

```diff
@@ -19,13 +19,19 @@ class ExternalRequestError(Exception):
     """
 
     def __init__(
-        self, message=None, request=None, response=None, validation_errors=None
-    ):
+        self,
+        message=None,
+        request=None,
+        response=None,
+        validation_errors=None,
+        refreshable=False,
```

Comment:
`ExternalRequestError.refreshable` has to be `False` by default because `ExternalRequestError` and its many subclasses (including `OAuth2TokenError`, `CanvasAPIError`, `CanvasAPIPermissionError`, `CanvasAPIServerError` and `HAPIError`) are raised all over the place, even in places like the `HAPI` client and the `LTIOutcomesService`. If `refreshable` were `True` by default we'd end up marking all sorts of things as refreshable